week-3-practice/exercise3.js

```diff
@@ -0,0 +1,12 @@
+/* 
+  You are given an array of test scores.
+  Any result less than 50 is a fail. 
+  Using .find() (and any other array methods necessary), find the lowest fail score
+*/
+var testScores = [49, 50, 100, 25, 81, 21, 80];
+var lowestPassScore = findLowestPassScore(testScores);
```

Comment:
This exercise too. All of your practice exercises won't work because you are missing the function